ui/selectors/assets.ts

```diff
@@ -1040,8 +1045,29 @@ export const getAssetsBySelectedAccountGroup = createDeepEqualSelector(
       ...multichainState,
     };
   },
-  (assetListState: AssetListState) =>
-    selectAssetsBySelectedAccountGroup(assetListState),
+  (assetListState: AssetListState) => {
+    const assetsByAccountGroup =
+      selectAssetsBySelectedAccountGroup(assetListState);
+
+    // Filter Tron tokens
+    Object.values(TrxScope).forEach((tronChainId) => {
+      assetsByAccountGroup[tronChainId] = assetsByAccountGroup[
+        tronChainId
+      ]?.filter((asset: Asset) => {
+        if (
+          asset.chainId.startsWith('tron:') &&
+          TRON_RESOURCE_SYMBOLS_SET.has(
+            asset.symbol?.toLowerCase() as TronResourceSymbol,
+          )
+        ) {
+          return false;
+        }
+        return true;
+      });
+    });
+
+    return assetsByAccountGroup;
```

Comment:
Discuss with @MetaMask/network-enablement team, can we move this TRON filtering into CORE (we do this in both mobile and extension)